esi_leap/api/controllers/v1/node.py

```diff
@@ -63,19 +64,26 @@ class NodesController(rest.RestController):
     @wsme_pecan.wsexpose(NodeCollection)
     def get_all(self):
         context = pecan.request.context
-        nodes = ironic.get_node_list(context)
 
-        node_collection = NodeCollection()
+        nodes = None
+        project_list = None
 
-        project_list = keystone.get_project_list()
-        now = datetime.now()
+        with concurrent.futures.ThreadPoolExecutor() as executor:
+            f1 = executor.submit(ironic.get_node_list, context)
+            f2 = executor.submit(keystone.get_project_list)
+            nodes = f1.result()
+            project_list = f2.result()
 
         offers = offer_obj.Offer.get_all({'status': [statuses.AVAILABLE]},
                                          context)
 
         leases = lease_obj.Lease.get_all({'status': [statuses.CREATED]},
                                          context)
 
+        node_collection = NodeCollection()
+
+        now = datetime.now()
```

Comment:
Is this a leftover debugging statement?

Reply:
It was there previously, just the position was different